doc/admin-guide/plugins/xdebug.en.rst

```diff
@@ -54,6 +54,12 @@ Diags
     transaction specific diagnostics for the transaction. This also requires
     that :ts:cv:`proxy.config.diags.debug.enabled` is set to ``1``.
 
+log-headers
+    If the ``log-headers`` is requested while :ts:cv:`proxy.config.diags.debug.tags` 
+    is set to ``xdebug.headers`` and :ts:cv:`proxy.config.diags.debug.enabled` is set to ``1``, 
+    then all client and server, request and response headers are logged. 
```

Comment:
Really, this only triggers if debug logging is enabled?

Reply:
yes. There was a concern of flooding the logs from outside request.